### PR TITLE
add tests for terminating SSL at LB to access console and servers

### PR DESCRIPTION
**Pull request by xiancao**
Add tests for terminating SSL at LB to access console and servers, including NGINX, Voyager and Traefik.

Jenkins result:
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/2638/
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/2643/
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/2645/

### CODE REVIEW

**Review by sankarpn — APPROVED** [review]
LGTM

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItTwoDomainsLoadBalancers.java [hunk 1]
```diff
@@ -1761,38 +1962,53 @@ private static void deployApplication(String namespace, String domainUid, String
...
+                                       String ingressHostName,
+                                       String pathLocation) {
+
+    StringBuffer consoleUrl = new StringBuffer();
+    if (isTLS) {
+      consoleUrl.append("https://");
+    } else {
+      consoleUrl.append("http://");
+    }
+    consoleUrl.append(K8S_NODEPORT_HOST)
```
Comment (anpanigr) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItTwoDomainsLoadBalancers.java:1977]: As we discussed, can u pass extra headers in curl command while accessing the console as follows ...
   -H "WL-Proxy-Client-IP: 120.1.1.0" -H "WL-Proxy-SSL: false"  and make sure that this header does not appear in  adminserver log. 

To get this header info you need to add the following DebugFlag on server startup args 

-Dweblogic.http.isWLProxyHeadersAccessible=true -Dweblogic.debug.DebugHttp=true -Dweblogic.StdoutDebugEnabled=true

Comment (xiancao) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItTwoDomainsLoadBalancers.java:1997]: fixed

**Review by anpanigr — APPROVED** [review]
LGTM

**Pull request merged**